CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ include_directories("third_party/METIS/include/")
 include_directories("third_party/dmlc-core/include")
 include_directories("third_party/minigun/minigun")
 include_directories("third_party/minigun/third_party/moderngpu/src")
+include_directories("third_party/cub/")
+include_directories("third_party/phmap/")
```

Comment:
we didn't include phmap before?

Reply:
Previous code uses relative include path like `include "../../third_party/phmap/parallel_hashmap/..."`. With this change, the include path is now a system one so you could use `include <parallel_hashmap/...>`.